tmt/steps/report/reportportal.py

```diff
@@ -0,0 +1,89 @@
+import dataclasses
+import io
+import os
+import zipfile
+from typing import List, Optional
+
+import click
+import junit_xml
+
+import tmt.steps.report
+import tmt.utils
+
+from . import junit
+
+
+@dataclasses.dataclass
+class ReportPortalData(tmt.steps.report.ReportStepData):
+    url: Optional[str] = None
+    token: Optional[str] = None
+    launch_name: Optional[str] = None
+
+
+@tmt.steps.provides_method("reportportal")
+class ReportPortal(tmt.steps.report.ReportPlugin):
+    """
+    Send results in JUnit format to Report Portal instance.
+    """
+
+    _data_class = ReportPortalData
+
+    @classmethod
+    def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecoratorType]:
+        """ Prepare command line options """
+        return [
+            click.option(
+                "--url", envvar="TMT_REPORT_REPORTPORTAL_URL", metavar="URL", required=True,
+                help="The URL of Report Portal instance where the data should be sent to"),
```

Comment:
```suggestion
                help="The URL of the Report Portal instance where the data should be sent to."),
```

Reply:
also there seems to be a typo here? `TMT_REPORT_REPORTPORTAL_URL`

Reply:
@thrix, what typo do you mean?

Reply:
Firstly, it looked there is duplicate REPORT, but I guess that was just me not understanding this is the reportportal report plugin :)